.github/CHANGELOG.md

```diff
@@ -2,6 +2,30 @@
 
 <h3>New features since last release</h3>
 
+* A new  `qml.make_drawing` method was added. This allows for qnodes to easily be drawn by simply giving an example input.
```

Comment:
Just to understand, is the name `make_drawing` chosen over the simpler `draw` because the function produces a kind of drawer, not a drawing? Would `drawer` then be shorter and more precise? Or `circuit_drawer`? I find `make_drawing` sounds a bit awkward...

Reply:
:+1:

Compare also to `qml.grad()` (or `qml.jacobian()`). These produce a gradient _function_, not a numeric value, but calling them `grad` and `jacobian` (rather than `make_grad_fn` or `make_grad`) hasn't been an issue/caused any confusion.

Reply:
How about `make_drawing(circuit) -> drawing(circuit)` or `drawer(circuit)`?

Reply:
I like `drawer`!

Reply:
me too, prefer `drawer`

Reply:
drawer it is then!

Reply:
How come `qml.draw` isn't getting any fans?

Reply:
too similar to `qnode.draw`?

Reply:
I also like `qml.draw()` 🤔

Reply:
The issue is that `qnode.draw` returns a string, where as `qml.drawer(circuit)` returns another function. I want to differentiate the two.